source/help/apps/desktop-mvp.rst

```diff
@@ -0,0 +1,9 @@
+**The following awards have been given for significant contributions to the Mattermost Desktop App:**
```

Comment:
Propose making this one credits page for all Mattermost contributors and having a section for Desktop App? 

Perhaps changing file name to contributors.rst?